cluster.tf

```diff
@@ -51,56 +69,49 @@ resource "google_container_cluster" "primary" {
   }
 
   addons_config {
+    # This is the default value but we want to be sure it will be always enabled
+    # This property will automatically create node pools based on resources consumption
+    horizontal_pod_autoscaling {
```

Comment:
I would say that since you are introducing this default here, make a variable out of it with the default value as "false".